.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: node_js
+
+stages:
+  - name: test
+    if: tag IS blank
+  - name: deploy
+    if: branch = master AND type != pull_request
+
+notifications:
+  email: false
+
+jobs:
+  include:
+    - stage: test
+      node_js: '9'
+      after_success:
+        - npm install coveralls@^2.11.9 && cat ./coverage/lcov.info | coveralls
```

Comment:
I'd change to `codecov`, they can merge the coverage better. We can discuss this matter and I show you what I actually mean.